### PR TITLE
DROOLS-2300: Inline controls: Invocation editor

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2300

Depends on #1453

I will rebase this once #1453 is merged.

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@jomarko This PR can be reviewed now. Thanks.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/resources/org/kie/workbench/common/dmn/client/resources/i18n/DMNClientConstants.properties [hunk 1]
```diff
@@ -32,8 +32,10 @@ RelationEditor.DeleteRow=Delete row
 RelationEditor.InsertColumnBefore=Insert column before
 RelationEditor.InsertColumnAfter=Insert column after
 RelationEditor.DeleteColumn=Delete column
+InvocationEditor.InsertParameterAbove=Add Parameter above
+InvocationEditor.InsertParameterBelow=Add Parameter below
```
[the remark this answers is not in the document]
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/resources/org/kie/workbench/common/dmn/client/resources/i18n/DMNClientConstants.properties]: Updated to be consistent (in new commit).

**Pull request merged**